dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/data/upload/UploadStatus.kt

```diff
@@ -125,10 +136,20 @@ internal sealed class UploadStatus(
                         "and you're targeting the relevant Datadog site."
                 )
             }
+            if (this@UploadStatus.shouldRetry) {
+                append(
```

Comment:
should add a space in the beginning in case if there is already some text, otherwise it will be like `...targeting the relevant Datadog site.Attempt count was...`